src/tags/catalog.tag

```diff
@@ -62,6 +62,9 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
         registryUI.catalog.repositories = [];
       });
       oReq.addEventListener('loadend', function() {
+        if (!registryUI.url()) {
+          registryUI._url = window.location.origin + window.location.pathname.replace(/\/+$/, '')
```

Comment:
This is usefull only for the **standard interface** right ? Why don't you put this in `script.js` ?
In this line :
https://github.com/Joxit/docker-registry-ui/blob/36c44b19a3c9be6a80c007b8121a2145472be2a3/src/scripts/script.js#L32-L33

The job is already done for the **static interface** in `static.js`
https://github.com/Joxit/docker-registry-ui/blob/36c44b19a3c9be6a80c007b8121a2145472be2a3/src/scripts/static.js#L18-L24

Reply:
Hi,

thank you for your feedback!

I would rather keep the UI in an error state until we get a successful response.

E.g. `Repositories of { registryUI.name() }` filled with the origin/path would not make sense for a UI-only domain.

Reply:
Hum ok, but `loadend` event is triggered even if there is an error. This should be done if we get a status code 200?

Reply:
Good spot!

I will move this up into the 'load' event into the `this.status == 200` branch.